lib/static/modules/reducer.js

```diff
@@ -375,3 +381,22 @@ function forceUpdateSuiteData(suites, test) {
     const id = getSuiteId(test);
     suites[id] = cloneDeep(suites[id]);
 }
+
+function withBrowserStorage(reducer) {
+    return (state, action) => {
+        const newState = reducer(state, action);
+
+        if (/^VIEW_/.test(action.type)) {
+            localStorageWrapper.setItem('view', {
+                showSkipped: newState.view.showSkipped,
+                showOnlyDiff: newState.view.showOnlyDiff,
+                scaleImages: newState.view.scaleImages,
+                lazyLoadOffset: newState.view.lazyLoadOffset,
+                groupByError: newState.view.groupByError,
+                testNameFilter: newState.view.testNameFilter
```

Comment:
With destructuring it will be a little shorter:
```
const {view} = reducer(state, action);
...
showSkipped: view.showSkipped,
showOnlyDiff: view.showOnlyDiff,
...
```

Reply:
`const {view} = reducer(state, action);` так нельзя, мы должны вернуть результат применения редьюсера из декоратора. Можно `const {view} = newState` но в этом смысла нет, лишняя переменная.